posts/2021-08-27_oban/content.html

```diff
@@ -0,0 +1,39 @@
+  
+<p>Just as we were getting ready to sleep, a sailing boat came along and anchored right in front of us, providing idyllic photos, but also making it quite a bit more work to go for a quick pee. </p>
+
+<p><img src="img/image06.jpg" /></p>
+<p><img src="img/image07.jpg" /></p>
+
+<p>After a good night of sleep, it was time to head back early because it was racing day in Oban. The eagle was not yet on his post, and we paddled calmly back to the car from where it was a few more kms drive into Oban. Right in front of Oban is the island Kerrera, and this was the scene for a kayak race with surf skis and touring kayaks. I was going to join, while Leigh was going to support and then shop in Oban when I would disappear from view. The race route was easy, keep Kerrera on your right and stay out of the way of ferries, until back in the harbour, a loop of 20 km. No problem. After a count down, off we went. </p>
+  
+<p><img src="img/image08.jpg" /></p> 
+
+<p> I’ve never really raced before in a kayak, the principles seemed simple: paddle as hard as you can over as little as possible distance as you can. But I soon found out there is more to consider. Try to be smart with energy, keep hydrated (it was a warm sunny day) and get your skeg magagement right. I have a skeg that I can adjust, and when it’s down my boat goes straighter but there is more water resistance, whereas when I have it up (not in the water) it is sometimes difficult to keep straight and I have to cover more distance because I wobble due to currents around rocks for example, which slows me down. But, there is less water resistance. While I was racing I must have adjusted my skeg about 50 times while I tried to go as close to the rocks of Kerrera as seemed sensible and cover as little distance as possible. Also to consider was the tidal current, when going in the same direction as the tide it might be better to be a bit further away from the shore because there is a stronger tidal current there, or there could be a back eddy near the rocks that slows me down. So sometimes the longest way is the shortest way, if you get what I mean.. No? I didn’t get it all the time either, but I managed to overtake a few people, and got overtaken by a few as well. I finished in a time I was happy with, and I always find it fun to race, so had a great time.</p>
+
+<p>I expected to be a lot slower then I was, so when I messaged Leigh that I was already finished, he was still in Oban in line for a famous crab sandwich and completely missed my sprint to the finish line. Not a problem, it was nice and sunny, and when Leigh finally got his sandwich I had already accepted my first price in the women's touring category, the Oban lifeboat cup (proceeds of the race went to the lifeboat). I have to admit, that I also came in last in my category, because apparently only men like to race with their kayaks. Oh well, I was happy with my time anyway, and had some nice chats with fellow paddlers. </p>
```

Comment:
```suggestion
<p>I expected to be a lot slower then I was, so when I messaged Leigh that I was already finished, he was still in Oban in line for a famous crab sandwich and completely missed my sprint to the finish line. Not a problem, it was nice and sunny, and when Leigh finally got his sandwich I had already accepted my first prize in the women's touring category, the Oban lifeboat cup (proceeds of the race went to the lifeboat). I have to admit, that I also came in last in my category, because apparently only men like to race with their kayaks. Oh well, I was happy with my time anyway, and had some nice chats with fellow paddlers. </p>
```